### ISSUE TITLE
None referrers issue

### ISSUE THREAD

**Issue by Maxime-J** [umami-software/umami#2076]
### Describe the Bug

With MySQL (possibly with PostgreSQL too), Umami v2.2.0 and dev,
(None) referrers are not shown anymore in dashboards (for pageviews registered on v2).

**Details:**

In db, `referrer_domain` is NULL for pageviews without referrer,
and in the db query (from [queries/analytics/pageview/getPageviewMetrics.ts](https://github.com/umami-software/umami/blob/master/queries/analytics/pageview/getPageviewMetrics.ts)), these null entries are left out by the comparison made to filter the website domain:

```
select referrer_domain x, count(*) y
from website_event
  
where website_event.website_id = ?
  and website_event.created_at >= ?
  and website_event.created_at between ? and ?
  and event_type = 1
  and website_event.referrer_domain != -- website domain
  
group by 1
order by 2 desc
limit 100
```

So they don't make it to the front.

### Database

MySQL

**Comment by franciscao633** [comment]
I believe this is the intended behavior for the referrers section. We would only display counts from actual external referrers, and filter out null records.

**Comment by Maxime-J** [comment]
Hi, this has been introduced in 1.34 https://github.com/umami-software/umami/releases/tag/v1.34.0
Not seeing that from pageviews registered on v2 surprised me, that's why I dug.

**Comment by mikecao** [comment]
Fixed in v2.4.0.
